rocky/tools/forms/boefje.py

```diff
@@ -36,6 +32,13 @@ class BoefjeAddForm(BaseRockyForm):
         widget=forms.Textarea(attrs={"rows": 3}),
         help_text=BOEFJE_DESCRIPTION_HELP_TEXT,
     )
+    interval = forms.CharField(
+        required=False,
+        label=_("Interval"),
+        widget=forms.TextInput(
+            attrs={"description": "Specify the timeout interval for tasks using this Boefje in minutes"}
```

Comment:
Missing translation maybe?
```suggestion
            attrs={"description": _("Specify the timeout interval for tasks using this Boefje in minutes")}
```